### DISCUSSION TITLE
Question: How to add suggestion popups on mention and hashtag button click.

### DISCUSSION THREAD

**Discussion by Sachin-chaurasiya** [github/markdown-toolbar-element#58] [Q&A] [answered]

**Comment by keithamus** [comment, marked as answer]
We use a different element for that, which is this one: https://github.com/github/text-expander-element

**Reply by Sachin-chaurasiya** [reply]
@keithamus , Can I use [text-expander-element](https://github.com/github/text-expander-element) and [markdown-toolbar-element](https://github.com/github/markdown-toolbar-element) together?.

**Reply by keithamus** [reply]
Yes definitely! This is exactly what we do on github.com

**Reply by Sachin-chaurasiya** [reply]
@keithamus , Can you please help me out here https://github.com/github/markdown-toolbar-element/discussions/60
